kuka_kr5_support/config/kuka_6dof_controllers.yaml

```diff
@@ -0,0 +1,51 @@
+controller_manager:
```

Comment:
maybe we should add this file into `kuka_resources` since all robots will use the same exemplary setup.

I see this was missing from ROS 1, but it was annoying… So, we should have a simple example with controller setup.